client/views/room/contextualBar/Info/EditRoomInfo/EditRoomInfo.js

```diff
@@ -374,7 +374,9 @@ function EditChannel({ room, onClickClose, onClickBack }) {
 						</Field.Row>
 					</Box>
 					<Field.Row>
-						<MultiSelect options={sysMesOptions} disabled={!hideSysMes} value={systemMessages} onChange={handleSystemMessages} placeholder={t('Select_an_option')} flexGrow={1}/>
+						<VerticalBar.ScrollableContent>
```

Comment:
I don't think this is the right place for a ScrollableContent. You should play with the element's css properties a bit, as I'm sure this can be fixed via CSS.